src/Compilers/CSharp/Portable/Binder/Binder_Conversions.cs

```diff
@@ -97,6 +98,11 @@ internal partial class Binder
                 return CreateTupleLiteralConversion(syntax, (BoundTupleLiteral)source, conversion, isCast, destination, diagnostics);
             }
 
+            if (conversion.Kind == ConversionKind.ImplicitDefault)
+            {
+                return new BoundDefaultOperator(source.Syntax, destination);
+            }
```

Comment:
I think `default` should be handled similarly to the way `null` is handled.

---
In reply to: [77467548](https://github.com/dotnet/roslyn/pull/13603#discussion_r77467548) [](ancestors = 77467548)